README.md

```diff
@@ -1 +1,93 @@
-# Disney-Characters
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Fruitycee
+
+A web application for a provides data for all kinds of fruit.
+
+# Live Demo
+- Add Allow CORS Extension for the api to work.
+- Please click on Logo in the site to make it work.
+
+[Live Demo](https://muskan2532150.github.io/Fruitycee)
+
+## Built With
+
+- HTML5
+- CSS3
+- ES6
+- React
+- Redux
+- React-Jest
+- Api
+- Create-react-App
+
+## Project Setup
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+- You need to have Code Editor Installed
+- Make sure Nodejs and NPM is installed in your system for linters and libraries
+
+### Setup
+
+1. Navigate to the folder where you want this cloned
+2. Clone the repo in your local machine:
+
+```bash
+git@github.com:muskan2532150/Fruitycee.git
+```
+
+3. Open terminal and cd into project and install the dependencies
+
+```bash
+cd Space-Travelers-Hub
+
```

Comment:
Please correct this. This is a metrics app project please kindly add the appropriate information for this project